usage-guides/assets.md

```diff
@@ -17,6 +17,28 @@ To solve this problem we added a whitelist which you can add assets to as can be
 
 ![Whitelist asset](/images/asset_whitelist_menu.png)
 
+## Token Detection Methods
+
+There are several ways to detect tokens:
+
+**Single Account Per-Chain Detection**
+
+The best way to detect tokens for a single address is to go to blockchain & accounts balances, scroll down to the EVM accounts vuew, expand the dropdown at the account, go to the per chain tab and click on the detect tokens toggle 🔃.
+
+![Example of detecting tokens for a specific chain](/images/redetect-per-chain-view.png)
+
+**All Accounts Per-Chain Detection**
+
+Go to Account & Balances -> Blockchain Balances. Scroll down to EVM accounts and click on Re-detect tokens (🔃). Here you can select specific chains to detect tokens for all your wallets.
```

Comment:
```suggestion
Go to Account & Balances -> Blockchain Balances. Scroll down to EVM accounts and click on Re-detect tokens (🔃). Here you can select specific chains to detect tokens for all your wallets. This will take a bit of time depending on tracked addresses per chain. The slowest chain for detection is ethereum.
```